v2_migrations/2024-08-23-112510_payment_methods_v2_db_changes/down.sql

```diff
@@ -0,0 +1,44 @@
+-- This file should undo anything in `up.sql`
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS accepted_currency "Currency"[];
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS scheme VARCHAR(32);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS token VARCHAR(128);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS cardholder_name VARCHAR(255);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS issuer_name VARCHAR(64);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS issuer_country VARCHAR(64);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS is_stored BOOLEAN;
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS direct_debit_token VARCHAR(128);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS swift_code VARCHAR(32);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS payment_method_issuer VARCHAR(128);
+
+CREATE TYPE "PaymentMethodIssuerCode" AS ENUM (
+    'jp_hdfc',
+    'jp_icici',
+    'jp_googlepay',
+    'jp_applepay',
+    'jp_phonepe',
+    'jp_wechat',
+    'jp_sofort',
+    'jp_giropay',
+    'jp_sepa',
+    'jp_bacs'
+);
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS payment_method_issuer_code "PaymentMethodIssuerCode";
+
+ALTER TABLE payment_methods DROP CONSTRAINT IF EXISTS payment_methods_pkey;
+
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS payment_method_id VARCHAR(64) PRIMARY KEY;
+
+UPDATE payment_methods SET payment_method_id = id;
```

Comment:
Same applies here, we'd have to copy over data to the `payment_method_id` column first and then add the primary key constraint.